src/content/facts/12-de-montfort.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Giant octopuses really aren't a thing.
+source: Wikipedia
+sourceUrl: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
+image: /images/colossal-octopus.jpg
+imageSource: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
+---
+
+Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. And you know how, when you think about something a whole lot, you start seeing it everywhere? Yep, that's what happened here.
```

Comment:
On first pass this sentence read a bit awkwardly to me. Here's my suggestion to tighten it up a little:

```suggestion
Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. You know how thinking about something a lot, you start seeing it everywhere? Yup, that's what happened here.
```